jenkins-pipelines/master-triggers/sct_triggers/weekly-master-rolling-upgrades-trigger.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?><project>
+  <actions/>
+  <description>Triggers Master Tests on weekly basis every Saturday at 6:00&#13;
+&#13;
+UPDATE 02/02/2023: Removed rolling-upgrade-multi-dc-ami-test and rolling-upgrade-with-sla-no-shares-test + added the rolling-upgrade-azure-image-test under Roy's request&#13;
```

Comment:
isn't it outdated? Does it matters?

Reply:
I don't know, that's the comments that was there, and I dumped it as is.

it's still true,  I.e. test mentioned are still removed, and the one added is still there.